api-reference/10 UI Widgets/dxDateBox/1 Configuration/applyValueMode.md

```diff
@@ -5,10 +5,10 @@
 The option accepts the following values.
 
 - "useButtons"  
- Enables an end-user to apply the selected value or cancel the selection using the Apply and Cancel buttons respectively.
+ Enables an end-user to apply the selected value or cancel the selection using the Apply and Cancel buttons respectively. When the [pickerType](/api-reference/10%20UI%20Widgets/dxDateBox/1%20Configuration/pickerType.md '/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#pickerType') is *'calendar'*, choosing this mode activates the [showTodayButton](/api-reference/10%20UI%20Widgets/dxCalendar/1%20Configuration/showTodayButton.md '/Documentation/ApiReference/UI_Widgets/dxCalendar/Configuration/#showTodayButton') option, and the Today button appears.
```

Comment:
From your writing, it is not clear how the Today button appearing is related to the rest of the sentence. Please rewrite.